Library/Formula/imply.rb

```diff
@@ -0,0 +1,21 @@
+class Imply < Formula
+  desc "Exploratory Analytics Powered By Druid"
+  homepage "http://imply.io/"
+  url "http://static.imply.io/release/imply-#{version}.tar.gz"
```

Comment:
We don't support this syntax on main URLs, and don't really want to. Please just inline it.